onnx/src/main/java/org/bytedeco/javacpp/onnx.java

```diff
@@ -1194,7 +1192,7 @@ private native void allocate(
   // Get output formal parameters.
   public native @Const @ByRef FormalParameterVector outputs();
 
-  public native @Const @ByRef TypeConstraintParamVector typeConstraintParams();
+  public native @StdVector TypeConstraintParam typeConstraintParams();
```

Comment:
Why does this no longer return a vector (TypeConstraintParamVector) ? This type change breaks my code.